### PR TITLE
Add autocompletion forwarding for wrappers

### PR DESCRIPTION
**Pull request by tucksaun**
Requires https://github.com/symfony-cli/console/pull/16 and https://github.com/symfony-cli/console/pull/15

Closes https://github.com/symfony-cli/console/pull/14

### CODE REVIEW

**Comment by fabpot** [comment]
@tucksaun Can you rebase to make the tests pass?

File: commands/wrappers.go [hunk 1]
```diff
@@ -27,24 +27,30 @@ import (
...
+		Aliases: []*console.Alias{{Name: "composer"}},
 		Action: func(c *console.Context) error {
 			return console.IncorrectUsageError{ParentError: errors.New(`This command can only be run as "symfony composer"`)}
 		},
 	}
 	binConsoleWrapper = &console.Command{
-		Name:   "console",
 		Usage:  "Runs the Symfony Console (bin/console) for current project",
 		Hidden: console.Hide,
+		// we use an alias to avoid the command being shown in the help but
```
Comment (fabpot) [inline commands/wrappers.go:42]: It's hidden anyway, no?

Comment (tucksaun) [reply commands/wrappers.go:42]: "yes... but"

with https://github.com/symfony-cli/console/pull/17, the alias will be considered for autocompletion. which means `symfony co` + tab will now suggest `symfony console`

**Comment by tucksaun** [comment]
@fabpot I think tests are broken for another reason, I'm having a look

**Comment by tucksaun** [comment]
@fabpot https://github.com/symfony-cli/console/pull/18 should fix tests

**Pull request merged**